### PR TITLE
🧑‍💻(api) enforce the declaration of 'verb_id' in all video indicators

### PR DESCRIPTION
**Pull request by lebaudantoine**
Added a missing check while constructing subclasses of `BaseDailyEvent`.
Improved the documentation on declaring new indicators that inherit from `BaseDailyEvent`.

These two additions should enhance the DX while using `BaseDailyEvent`.

### CODE REVIEW

File: src/api/plugins/video/tests/test_indicators.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""Tests for the video Indicators."""
+
+import pytest
+from warren_video.indicators import BaseDailyEvent
+
+
+@pytest.mark.anyio
+async def test_base_daily_event_verb_id():
+    """Test __init_subclass__ for the BaseDailyEvent class."""
```
Comment (lebaudantoine) [inline src/api/plugins/video/tests/test_indicators.py:9]: nit: make these naming coherent

File: src/api/plugins/video/tests/test_indicators.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+"""Tests for the video Indicators."""
```
Comment (lebaudantoine) [inline src/api/plugins/video/tests/test_indicators.py:1]:
```suggestion
"""Tests for the video indicators."""
```

**Review by jmaupetit — APPROVED** [review]

**Review by quitterie-lcs — APPROVED** [review]

**Pull request merged**